src/rows.js

```diff
@@ -68,6 +68,24 @@ export class Rows {
     return { value: row, done: false };
   }
 
+  /**
+   * Rows.columns
+   *
+   * Call this if you need column names from the result of a select query.
```

Comment:
Add a comment about the returned objects shape (i.e. that it exposes name, originName, and tableName) and maybe mention which sqlite functions they correspond to (e.g. in a table).